querydsl-codegen/src/main/java/com/querydsl/codegen/CapitalizedVariableNameFunction.java

```diff
@@ -0,0 +1,31 @@
+package com.querydsl.codegen;
+
+import com.google.common.base.Function;
+import com.mysema.codegen.StringUtils;
+import com.querydsl.core.util.JavaSyntaxUtils;
+
+/**
+ * Variable name generation strategy which capitalizes the first letter of the variable name.
+ *
+ */
+public class CapitalizedVariableNameFunction implements Function<EntityType, String> {
```

Comment:
If you yourself have your `com.your.package.name.VariableNameFunction`, I wouldn't really see a reason to include this file.
We'll then also have to provide support for it.